pkg/clusteragent/admission/mutate/autoinstrumentation/injector.go

```diff
@@ -196,7 +196,7 @@ func injectorDebug(boolean string) injectorOption {
 	}
 }
 
-func newInjector(startTime time.Time, registry, imageTag string, opts ...injectorOption) *injector {
+func newInjector(startTime time.Time, registry string, opts ...injectorOption) *injector {
```

Comment:
Should registry and startTime also be opts? 🤔

Reply:
Thinking about dropping `newInjector` entirely but that might be for another follow up. Thanks for the review.